### PR TITLE
Update CommonUtil.java

### PR DESCRIPTION
**Pull request by berkant-k**
When logLevel set to FINE;  castings to URLClassLoader  was throwing  java.lang.ClassCastException.
To fix this issue logClassPath method changed

Signed-off-by: Berkant Karduman <karduman.berkant@gmail.com>

### CODE REVIEW

File: fhir-persistence-schema/src/main/java/org/linuxforhealth/fhir/schema/app/util/CommonUtil.java [hunk 1]
```diff
@@ -62,11 +62,10 @@ public static void configureLogger() {
     public static void logClasspath(Logger logger) {
         if (logger.isLoggable(Level.FINE)) {
             logger.fine("CLASSPATH: ");
-            ClassLoader cl = ClassLoader.getSystemClassLoader();
-            URL[] classpath = ((URLClassLoader) cl).getURLs();
-            for (URL u : classpath) {
-                logger.fine("  " + u.getFile());
-            }
+            String[] classPaths = System.getProperty("java.class.path").split(File.pathSeparator);
```
Comment (lmsurpre) [inline fhir-persistence-schema/src/main/java/org/linuxforhealth/fhir/schema/app/util/CommonUtil.java:65]: is `System.getProperty("java.class.path")` a reliable way to get the classPath string?

is it always delimited by File.pathSeparator?  I might have expected ":" for *nix, and ";" for windows ... is the `java.class.path` documented somewhere?

Comment (lmsurpre) [reply fhir-persistence-schema/src/main/java/org/linuxforhealth/fhir/schema/app/util/CommonUtil.java:65]: why not just log the result of `System.getProperty("java.class.path")` directly?

Comment (berkant-k) [reply fhir-persistence-schema/src/main/java/org/linuxforhealth/fhir/schema/app/util/CommonUtil.java:65]: Yes. You are right. I will edit the code.

**Review by lmsurpre — APPROVED** [review]
LGTM. Thanks @berkant-k!

**Pull request merged**